src/packages/core/src/base-resolver.ts

```diff
@@ -91,6 +91,13 @@ export function createBaseResolver<G extends WithId, D extends BaseDataEntity>(
 	const entityFields = metadata.fields.filter((field) => field.target === gqlEntityType);
 	const enumSet = new Set(metadata.enums.map((enumMetadata) => enumMetadata.enumObj));
 
+	const entityMetadata = {
```

Comment:
is this being used? Could you use it inside both the `EntityMetadataMap` and for the `static metadata`?

Reply:
This could be `const entityMetadata: BaseResolverMetadataEntry<D>` then you can remove the cast on line 100?